process/interface.go

```diff
@@ -349,9 +349,9 @@ type BlockSizeThrottler interface {
 	IsInterfaceNil() bool
 }
 
-// TxValidatorHandler define the functionality that is needed for a TxValidator to validate a transaction
+// TxValidatorHandler defines the functionality that is needed for a TxValidator to validate a transaction
 type TxValidatorHandler interface {
-	GetSenderShardId() uint32
+	SenderShardId() uint32
 	GetNonce() uint64
```

Comment:
Should also drop the Get func names from the rest of the interface

Reply:
Done.